docs/content/posts/2023-10-10-release-1.18.0.md

```diff
@@ -0,0 +1,572 @@
+parallelism have been acquired. The rescaling operation is not based on a Savepoint, but on an ordinary, periodic 
+checkpoint, which means it does not introduce any additional snapshot. As you can see in the video above, the rescaling
+operation already happens nearly instantaneously and with a very short downtime for jobs with small state size.
+
+In conjunction with the 
+[backpressure monitor](https://nightlies.apache.org/flink/flink-docs-stable/docs/ops/monitoring/back_pressure/) 
+of the Apache Flink Web UI, it is now easier than ever to find and maintain an efficient, backpressure-free parallelism 
+for each of the tasks:
+
+- If a task is very busy (red), you increase the parallelism.
+- If a task is mostly idle (blue), you decrease the parallelism.
+
+<div style="text-align: center;">
+<img src="/img/blog/2023-10-10-release-1.18.0/backpressure_monitor.png" style="width:90%;margin:15px">
+</div>
+
+**More Information**
+* [Documentation](missing)
+* [FLIP-291: Externalized Declarative Resource Management](https://cwiki.apache.org/confluence/display/FLINK/FLIP-291%3A+Externalized+Declarative+Resource+Management)
+
+## Faster Rescaling with RocksDB
+
+The rescaling times when using RocksDB Statebackend with incremental checkpoints have been improved about 30% in the 99th quantile.
+
+We increased the potential for parallel download from just downloading state handles in parallel to downloading individual files in parallel.
+
+Furthermore, we deactivated write-ahead-logging for batch-inserting into the temporary RocksDB instances we use for rescaling.
+
+<div style="text-align: center;">
+<img src="/img/blog/2023-10-10-release-1.18.0/rescaling_performance.png" style="width:90%;margin:15px">
+</div>
+
+**More Information**
+* [FLINK-32326](https://issues.apache.org/jira/browse/FLINK-32326)
+* [FLINK-32345](https://issues.apache.org/jira/browse/FLINK-32345)
+
+# Support for Java 17 
+
+Java 17 was released in 2021 and is the latest long-term support (LTS) release of Java with an end-of-life in 2029. 
+So, it was about time that Apache Flink added support for it. What does this mean concretely? As of Flink 1.18, you can
+now run Apache Flink on Java 17 and the [official Docker repository](https://hub.docker.com/_/flink) includes an image 
+based on Java 17.
+
+    docker pull flink:1.18.0-java17
+
+If your cluster runs on Java 17, this of course, also allows you to use Java 17 features in your user programs and to
+compile it to a target version of Java 17.
+
+**More Information**
+* [Documentation](https://nightlies.apache.org/flink/flink-docs-stable/docs/deployment/java_compatibility/)
+* [FLINK-15736](https://issues.apache.org/jira/browse/FLINK-15736)
+
+# Others Improvements
+
+## Production-Ready Watermark Alignment
+
+Supported as “Beta” since Flink 1.16 and Flink 1.17, 
+[watermark alignment](https://nightlies.apache.org/flink/flink-docs-stable/docs/dev/datastream/event-time/generating_watermarks/#watermark-alignment) 
```

Comment:
I think the link should be https://nightlies.apache.org/flink/flink-docs-stable/docs/dev/datastream/event-time/generating_watermarks/#watermark-alignment-_beta_